aten/src/ATen/native/mps/operations/Indexing.mm

```diff
@@ -608,17 +608,20 @@ Tensor index_select_mps(const Tensor& self, int64_t dim, const Tensor& index) {
       newCachedGraph->outputTensor_ = outputTensor;
     });
 
+    // MPS TODO: MPS Gather is failing with MPS strided API. Fallback to old gather.
```

Comment:
There are a couple TODOs lying around, should we open issues to track them down?

Reply:
I will update with the issue number